README.org

```diff
@@ -32,6 +32,38 @@ the environment variable =ZMQ_VERSION=, e.g. to something like
 ZMQ_VERSION=4.3.0
 #+END_SRC
 
+*** Windows
+
+It is possible to use the included build chain on Windows using the [[https://www.msys2.org/][MSYS2]] MinGW
+tools.
+
+Install the 64-bit toolchain inside the MSYS2 shell via
+#+BEGIN_SRC shell
+pacman -S base-devel
+pacman -S git
+pacman -S mingw-w64-x86_64-gcc
+#+END_SRC
+
+Note: If you are using the official Git for Windows instead of MSYS2 Git, make
+sure to set
+
+#+BEGIN_SRC shell
+git config --global core.autocrlf false
+#+END_SRC
+
+during the build to avoid EOL issues and set it back to =true= (the default on
+Windows) when you are done building.
+
+Start the build from an MSYS2 MinGW 64-bit shell via =make=.
+
+The =libzmq.dll= is built to =libzmq/build/v*/bin= and needs to be copied to the
```

Comment:
I don't think we need this anymore since we copy over the `libzmq.dll` into the same directory as `emacs-zmq.dll` on Windows systems.

Reply:
True, removing it.